### ISSUE TITLE
Extracting history from best trained model and viewing progress

### ISSUE THREAD

**Issue by amjass12** [keras-team/keras-tuner#120]
Hello all, 

I was just wondering if there is a way of viewing the history of the best model and using this to plot loss and accuracy and metrics so we can visualise exactly how the model has progressed when selecting the best model?

I want to be sure that the model is not overfitting so visualising this would be very helpful. 

Many thanks!

Amir

**Comment by omalleyt12** [comment]
@amjass12 Thanks for the issue! If you pass in a `TensorBoard` callback to `search` like:

```python
tuner = ...
tuner.search(x, y, validation_data=(val_x, val_y),
       callbacks=[tf.keras.callbacks.TensorBoard('my_dir')])
```

you should see a bunch of subdirectories in TensorBoard, one for each trial with the same name is the trial_id. Find the trial_id of the best model like:

```
tuner.oracle.get_best_trials()[0].trial_id
```

and then look for its plots on TensorBoard (make sure you're using the `master` branch version)

We're also working with the TensorBoard team on native support for displaying the progress of the search (with best model found so far, etc), which should be a better UX, but that's not yet available

Please let me know if you run into any issues with this

**Comment by amjass12** [comment]
@omalleyt12 

This is brilliant!! thank you so much! and that answers my question perfectly! 👍 Thanks again! i look forward to native support!! that would be great!

**Comment by omalleyt12** [comment]
Great, thanks glad that helped!

**Comment by svdHero** [comment]
>
> and then look for its plots on TensorBoard (make sure you're using the `master` branch version)
> 
@omalleyt12 Are you talking about TensorBoard's `master` or KerasTuner's `master`?

When using Tensorflow 2.0.0 with TensorBoard 2.0.1 and Keras-Tuner 1.0.0, I get tonnes of error messages of the kind
```
[plugin_event_accumulator.py:294] Found more than one graph event per run, or there was a metagraph containing a graph_def, as well as one or more graph events.  Overwriting the graph with the newest event.
```
How do I fix this? Which version combination of the above is needed?

As a side-note: I am a Tensorflow beginner and kind of hesitant to download and compile Python packages from scratch. So `pip install` is my friend. :)

**Comment by 21kc-caracol** [comment]
So just to be certain: Is there a way to get the same behavior as in 
`history = model.fit()`?
This could be very useful for different calculations. I use the `history` for `sklearn`'s precision and recall functions...

Thanks!
